packages/strapi-admin/admin/src/components/Roles/Permissions/utils/createDefaultCTFormFromLayout.js

```diff
@@ -131,14 +131,19 @@ const createDefaultCTFormFromLayout = (
     // The object has the following shape: { [ctUID]: { [actionId]: { [property]: { enabled: false } } } }
     const contentTypesActions = Object.keys(subjectLayouts).reduce((acc, currentCTUID) => {
       const { actionId, applyToProperties } = current;
+      const currentSubjectLayout = subjectLayouts[currentCTUID];
+      const properties = currentSubjectLayout.properties.map(({ value }) => value);
+      const doesNothaveProperty = properties.every(
+        property => (applyToProperties || []).indexOf(property) === -1
```

Comment:
Aren't you missing the function call on `applyToProperties` ? Right now, it's a defined and well working function nope?